bootstrap/terraform/gcp-bootstrap/main.tf

```diff
@@ -63,6 +63,12 @@ module "gke" {
   release_channel            = var.release_channel
   regional                   = var.regional_cluster
   zones                      = var.cluster_zones
+  cluster_resource_labels    = merge(
```

Comment:
this can be updated in place right?

Reply:
Yes, it will be updated in-place

```

  # module.gcp-bootstrap.module.gke.google_container_cluster.primary will be updated in-place
  ~ resource "google_container_cluster" "primary" {
        id                          = "projects/XXX/locations/XXX/clusters/data-dev"
        name                        = "data-dev"
      ~ resource_labels             = {
          + "managed-by" = "plural"
        }
        # (27 unchanged attributes hidden)

        # (19 unchanged blocks hidden)
    }
```

Reply:
@michaeljguarino up 🙏